src/components/EmojiText/index.native.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import Text from '../Text';
+
+// As this workaround is not needed on the native platform, we simply render Text  
+const EmojiText = props => (
+    <Text {...props} />
+);
+
+EmojiText.displayName = 'EmojiText';
+
```

Comment:
Please add prop definitions that are used for Text component.

Reply:
I have updated the pull request